packages/@aws-cdk/aws-apprunner/test/integ.service-observability-configuration.ts

```diff
@@ -0,0 +1,34 @@
+import * as cdk from '@aws-cdk/core';
```

Comment:
I'm not totally sure how this didn't produce an error when you ran the test, but this test needs to use the `IntegTest` construct.